[DOCS] mention PostgreSQL extensions when upgrading (#6008)
yugabyte/yugabyte-db: docs/content/latest/manage/upgrade-deployment.md

4. ./bin/post_install.sh
```

{{< note title="Note" >}}

> If you are using PostgreSQL extensions, make sure to install them in the new version before upgrading. You can follow
>  the [same guide](../../api/ysql/extensions) that you installed the extensions in the first place. 

Reviewer: ```suggestion
If you are using PostgreSQL extensions, make sure to install the extensions in the new YugabyteDB version before upgrading. Follow the steps in [Install and use extensions](../../api/ysql/extensions). 
```
